DearPyGui/src/core/AppItems/textures/mvRawTexture.cpp

```diff
@@ -168,4 +185,77 @@ namespace Marvel {
 			return;
 	}
 
+	PyObject* mvRawTexture::get_raw_texture(PyObject* self, PyObject* args, PyObject* kwargs)
+	{
+
+		mvUUID item;
+
+		if (!(mvApp::GetApp()->getParsers())["get_raw_texture"].parse(args, kwargs, __FUNCTION__,
+			&item))
+			return GetPyNone();
+
+		if (!mvApp::s_manualMutexControl) std::lock_guard<std::mutex> lk(mvApp::s_mutex);
+
+		auto raw_texture = mvApp::GetApp()->getItemRegistry().getItem(item);
+		if (raw_texture == nullptr)
+		{
+			mvThrowPythonError(mvErrorCode::mvItemNotFound, "get_raw_texture",
+				"Item not found: " + std::to_string(item), nullptr);
+			return GetPyNone();
+		}
+
+		if (raw_texture->getType() == mvAppItemType::mvRawTexture)
+		{
+
+			auto pRawTexture = static_cast<mvRawTexture*>(raw_texture);
+			auto texture_id = (unsigned int)(size_t)pRawTexture->_texture;
+			return Py_BuildValue("k", texture_id);
+		}
+		else
+		{
+			mvThrowPythonError(mvErrorCode::mvIncompatibleType, "get_raw_texture",
+				"Incompatible type. Expected types include: mvRawTexture", raw_texture);
+		}
+
+		return GetPyNone();
+	}
+
+	PyObject* mvRawTexture::set_update_enable(PyObject* self, PyObject* args, PyObject* kwargs)
+	{
+
+		mvUUID item;
+		bool value;
```

Comment:
can you try changing this line to `int value;` Just that though.

Reply:
Wow it works!

Reply:
Awesome. The python C API is well... C lol. So they don't really have a **bool**. Instead they typically use the **BOOL** macro, which is an **int**. When using a C++ bool in their parsing commands it doesn't work right because the C++ bool is 1 byte, while an int is 4 bytes.